lib/phoenix/controller.ex

```diff
@@ -282,6 +282,21 @@ defmodule Phoenix.Controller do
   end
 
   @doc """
+  Sends js response.
+
+  ## Examples
+
+      iex> js conn, "console.log('Ground Control to Major Tom')"
+
+  """
+  @spec js(Plug.Conn.t, iodata) :: Plug.Conn.t
+  def js(conn, data) do
+    conn
+    |> put_resp_content_type(Plug.MIME.type("js"))
```

Comment:
Maybe we should just hardcode the value Plug.MIME returns here? Similar to how we do for `html/2`?